apps/sms/views/conversation/test/unit/link_helper_test.js

```diff
@@ -153,12 +153,14 @@ suite('link_helper_test.js', function() {
       test('non ascii prefix', function() {
         testURLMatch('àmozilla.org', 'mozilla.org', true);
       });
+      test('Bug 941763: Add dot as URL prefix', function() {
```

Comment:
Yeah it's good point that we actually don't need the bug # in this case. I think it's a bad practice from other test cases in this script that only left bug # in the title, so I suggest @ythej to add more details for the title.  So here is some ideas: We create another bug for polishing and remove all the unnecessary bug number in this test, or simply fix right in this patch. @ythej wdyt?